integr8ly/integreatly-operator
# Intly 6954 slo dashboards

# Description
The following PR is to show the changes made to create a SLO dashboard in grafana.  [Jira](https://issues.redhat.com/browse/INTLY-6594)


Verification
1. Install integreatly from this branch
2. make a few alerts fire
3. verify the dashboard portrays this change.

![Screenshot from 2020-04-08 17-56-32](https://user-images.githubusercontent.com/53817495/78811807-554dcd80-79c2-11ea-890d-3c23354c9fc5.png)
